playbooks/update_files.yml

```diff
@@ -146,6 +146,37 @@
             __present_python_templates + __present_shellcheck_templates)) +
             ([__role_template_dir] | product(__role_present_templates)) }}"
 
+        - name: Find workflows to create badges for
+          find:
+            paths: "{{ git_dir }}/.github/workflows"
+            file_type: file
+            excludes:
+              - ansible-managed-var-comment.yml
+              - ansible-plugin-scan.yml
+              - build_docs.yml
+              - changelog_to_tag.yml
+              - pr-title-lint.yml
+              - test_converting_readme.yml
+              - weekly_ci.yml
+          register: find
+
+        - name: Ensure badges in README.md
+          lineinfile:
+            path: "{{ git_dir }}/README.md"
+            regex: \[\!.*
+            firstmatch: true
+            line: "{%-for file in find.files -%}\
+              [![{{ file.path | basename }}]\
```

Comment:
This means the badge will have the filename instead of a "user friendly" name?

Reply:
Nope, GitHub renders this fine to the name of the workflow, and it's easier to get the filename than to read it's name from a file.